modules/eco-self-node-remediation-operator-troubleshooting.adoc

```diff
@@ -36,12 +36,22 @@ If the `MachineHealthCheck` controller did not create the `SelfNodeRemediation`
 If the `SelfNodeRemediation` CR was created, ensure that its name matches the unhealthy node or the machine object.
 
 [id="daemon-set-exists_{context}"]
-== Daemon set exists even after uninstalling the Self Node Remediation Operator
-Issue:: The self node remediation daemon set exists even after after uninstalling the Operator.
+== Daemon set and other Self Node Remediation Operator resources exist even after uninstalling the Operator
```

Comment:
Does this file need a doc type at the start?

Reply:
Yes, it does. Thank you. I have added it now